content/game-controllers/vkb/encoder-support/_index.md

```
@@ -0,0 +1,46 @@
+To circumvent this limit, MobiFlight can directly read the encoder position from the controller. For this to work, the controller must be configured in VKBDevCfg to make these positions available:
+
+{{% steps %}}
+
+### Close MobiFlight
+
+Close any running instance of MobiFlight Connector. Otherwise, you will need to restart it after changing the connected game controllers.
+
+### Verify controller firmware
+
+Ensure your controller firmware is updated to version 2.18.5 or newer. If not, download zBootloader and the firmware files from the [VKB website](https://www.vkbcontrollers.com/pages/downloads) and follow VKB's guides to upgrade your controller.
+### Access configuration in VKBDevCfg
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)